package.json

```diff
@@ -188,14 +188,14 @@
     "yup": "0.32.11"
   },
   "devDependencies": {
-    "@chainsafe/blst": "0.2.4",
     "@babel/core": "7.13.16",
     "@babel/plugin-proposal-nullish-coalescing-operator": "7.13.8",
     "@babel/plugin-proposal-optional-chaining": "7.13.12",
     "@babel/plugin-syntax-import-meta": "7.10.4",
     "@babel/plugin-transform-runtime": "7.13.15",
     "@babel/preset-env": "7.13.15",
     "@babel/preset-react": "7.13.13",
+    "@chainsafe/blst": "0.2.4",

```

Comment:
i dont see this package imported anywhere. Why did you installed it?

Reply:
its just change the order it's used by sdk v6